server/server.go

```diff
@@ -94,7 +94,10 @@ func (s *ConformanceServer) Check(ctx context.Context, in *exprpb.CheckRequest)
 
 // Eval implements ConformanceService.Eval.
 func (s *ConformanceServer) Eval(ctx context.Context, in *exprpb.EvalRequest) (*exprpb.EvalResponse, error) {
-	env, _ := cel.NewEnv(cel.Container(in.Container))
+	tp := types.NewProvider()
```

Comment:
Comment why we make this explicit.  Instead, perhaps add env.GetProvider() method?